### ISSUE TITLE
Fix worker config to mount the ephemeral storage correctly

### ISSUE THREAD

**Issue by jamierajewski** [jamierajewski/illume-v2#60]
Right now the storage isn't partitioned at all, but is instead sitting in a clump unused:
```bash
/dev/vdb1                                                   3.0T   89M  2.8T   1% /mnt
```
In reality, it should be split three ways as it is on the ingress:
```bash
/dev/vdb3                                                   141G  3.0G  131G   3% /var/lib/cvmfs
/dev/vdb2                                                   141G  121M  134G   1% /var/log/condor
/dev/vdb1                                                   1.2T   77M  1.1T   1% /scratch
```

**Comment by jamierajewski** [comment]
Likely due to using a volume to load and boot the image from, which reserves all of the ephemeral for partitioning (whereas in the workers, ephemeral is set for both booting and partitioning).

**Comment by jamierajewski** [comment]
First issue was some missing commas in the cloud-config, but with that fixed, now only the first partition is created (`ephemeral0` / `/scratch`). The boot log indicates that the special device names `ephemeral1` and `ephemeral2` could not be found, which is particularly strange given that the first one worked.

**Comment by jamierajewski** [comment]
This happened due to setting the partition map to 95 / 5 / 5 which erroneously tries to map 105% of the disk space. Rather than fail, it created one partition and mounted that to `/scratch`, hence why it worked but not the others. 

This only occurred in testing though, and the real partition map was actually correct. The true error that remained seemed to be one of two things:
- Incorrect formatting in the cloud-init section (addressed by b02a75d041e0a7ecb6535af50b9e64367be14cfb)
- Incorrectly parsing the size value from the flavor name (addressed by c9d15788a99f0873ed9298461eecc2e50e0f388a)
